lib/components/decorateMapComponent.js

```diff
@@ -32,7 +32,17 @@ export const createNotSupportedComponent = message => () => {
   return null;
 };
 
-export const googleMapIsInstalled = !!NativeModules.UIManager[getAirMapName(PROVIDER_GOOGLE)];
+function getViewManagerConfig(viewManagerName) {
+  const UIManager = NativeModules.UIManager;
+  if (!UIManager.getViewManagerConfig) {
```

Comment:
We have lodash in dependencies, so I think testing for a function is more accurate. And I'd put it the way around:
```
_.isFunction(UIManager.getViewManagerConfig) { // RN >= 0.58
  return UIManager.getViewManagerConfig(viewManagerName);
}
return return UIManager[viewManagerName]; // RN < 0.58

Reply:
Ok, seems reasonable